mono/utils/mono-error.c

```diff
@@ -100,49 +115,42 @@ mono_error_init_flags (MonoError *oerror, unsigned short flags)
 	error->flags = flags;
 }
 
+MONO_RT_EXTERNAL_ONLY void
```

Comment:
`MONO_RT_EXTERNAL_ONLY` should be in the `.h` and not the `.c`

Reply:
It makes it easier to read to put it in both but ok.

Reply:
Up